src/components/LinkedAccounts/LinkedAccounts.tsx

```diff
@@ -7,6 +7,7 @@ import { Loader } from '../Loader'
 import { Heading, HeadingSize } from '../Typography'
 import { LinkedAccountsContent } from './LinkedAccountsContent'
 import { OpeningBalanceModal } from './OpeningBalanceModal/OpeningBalanceModal'
+import { AccountConfirmationStoreProvider } from '@layerfi/components/providers/AccountConfirmationStoreProvider'
```

Comment:
I think this may break local development
```suggestion
import { AccountConfirmationStoreProvider } from '../../providers/AccountConfirmationStoreProvider'
```

Reply:
Apologies - I have changed one of my VSCode settings to stop it from looking in the wrong directory.